.circleci/config.yml

```diff
@@ -42,13 +43,20 @@ jobs:
             poetry run pytest -s --junitxml=test-results/junit.xml -vv $TESTFILES
       - store_test_results:
           path: test-results
+
+  test-py-max:
+    <<: *test-template
+    docker:
+      - image: cimg/python:3.9.12-node
```

Comment:
Wouldn't it be better to have it as `cimg/python:3.9-node`? That way, the latest patch version of 3.9 would be used

Reply:
Indeed, fixed :+1: